tools/setupHelpers.py

```diff
@@ -476,9 +476,57 @@ def getVersionStrings(pkg):
     return version, forcedVersion, gitVersion, initVersion
 
 
-from distutils.core import Command
-import shutil, subprocess
-from generateChangelog import generateDebianChangelog
+DEFAULT_ASV: Dict[str, Any] = {
+    "version": 1,
+    "project": "pyqtgraph",
+    "project_url": "http://pyqtgraph.org/",
+    "repo": ".",
+    "branches": ["master"],
+    "environment_type": "virtualenv",
+    "show_commit_url": "http://github.com/pyqtgraph/pyqtgraph/commit/",
+    # "pythons": ["3.7", "3.8", "3.9"],
+    "matrix": {
+        # "numpy": ["1.17", "1.18", "1.19", ""],
+        "numpy": "",
+        "pyqt5": ["", None],
+        "pyside2": ["", None],
+    },
+    "exclude": [
+        {"pyqt5": "", "pyside2": ""},
+        {"pyqt5": None, "pyside2": None}
+    ],
+    "benchmark_dir": "benchmarks",
+    "env_dir": ".asv/env",
+    "results_dir": ".asv/results",
+    "html_dir": ".asv/html",
+    "build_cache_size": 5
+}
+
+
+class ASVConfigCommand(Command):
+    description = "Setup the ASV benchmarking config for this system"
+    user_options = []
+
+    def initialize_options(self) -> None:
+        pass
+
+    def finalize_options(self) -> None:
+        pass
+
+    def run(self) -> None:
+        config = DEFAULT_ASV
+        try:
+            cuda_check = subprocess.check_output(["nvcc", "--version"])
+            match = re.search(r"release (\d\d.\d)", cuda_check.decode("utf-8"))
+            ver = match.groups()[0]  # e.g. 11.0
+            ver_str = ver[0] + ver[1] + ver[3]  # e.g. 110
```

Comment:
I apologize as you're walking into some issues I have with manipulating strings by their index position, which I personally try to avoid (not sure why) so feel free to disregard.

```python
ver_str = ver.replace(".", "", 1)  # the 1 at the end is for the number of times, alternatively, don't bound it, and just have `ver.replace(".", "")`
```